### PR TITLE
Website redesign 2021

### PR DESCRIPTION
**Pull request by codehag**
here is what it looks like:

Screenshot 2021-04-26 at 19 12 56

There is still a lot to be done. Tuning, getting mobile right, getting light/dark mode right, getting it working across all browsers -- making it pretty. but this is enough that we can collaborate together.

### CODE REVIEW

File: _sass/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
+// Colors
+$color-lightmode-primary: orange;
```
Comment (codehag) [inline _sass/_variables.scss]: initial stab at "modes"

**Comment by codehag** [comment]
there is even a favicon!

**Comment by codehag** [comment]
also, initial shoddy mobile view:

File: _layouts/default.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <section class="main-content">
+    <div class="g-row">
+      <div class="g-col-9">
+      {{ content }}
+      </div>
+    </div>
+    </section>
+    </main>
+    {% include footer.html %}
+    <img class="hidden" src="https://d3nrsflo8ae6b5.cloudfront.net/1x1.png" alt="pixel" />
```
Comment (moztcampbell) [inline _layouts/default.html]: What is this for?

File: index.markdown [hunk 3]
```diff
@@ -1,44 +1,11 @@
 ---
-layout: default
+layout: home
 title: Home
```
Comment (moztcampbell) [inline index.markdown:3]: lower-case "home" for the title for consistency

File: _layouts/default.html (hunk 2 above)
Comment (codehag) [reply _layouts/default.html]: ah, tracking. wrong tracking

File: index.markdown (hunk 3 above)
Comment (moztcampbell) [reply index.markdown:3]: Never mind, you fixed the others!

Comment (codehag) [reply index.markdown:3]: I think they are all upper now

**Review by moztcampbell — APPROVED** [review]
Update the title on the PR, but this looks great!

**Comment by codehag** [comment]
squishy

**Pull request merged**